.github/workflows/build-core.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         if: matrix.os == 'macos-latest'
 
       # Build and test TypeScript
-      - run: yarn --immutable --immutable-cache
+      - run: yarn --immutable --immutable-cache --ignore-engines
```

Comment:
aoconnect requires this 😢 - they rec npm over yarn so installs fail

Reply:
sad...